capstone/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "package.json-mastery",
+  "version": "1.0.0",
+  "scripts": {
+    "installTS": "npm install -g typescript",
```

Comment:
If you leave out the -g option does this still work?  If so I think it might be better not to include -g as that will install typescript globally on the users machine which may interfere with other work they do.  Just doing `npm install typescript` should only install it for use in this one project which I think is preferable

Reply:
I've made this change in "modify typescript download script"